terra-jupyter-base/jupyter_notebook_config.py

```diff
@@ -0,0 +1,51 @@
+# adapted from https://github.com/jupyter/docker-stacks/blob/master/base-notebook/jupyter_notebook_config.py
+
+from jupyter_core.paths import jupyter_data_dir
```

Comment:
is this only needed for running it locally? if so, can we add a comment about it

Reply:
This was here before… actually I'm not sure what it's needed for. Maybe I'll try removing it.

Reply:
Same for all these imports, except `os`